product_docs/docs/migration_portal/4/03_mp_using_portal/mp_ai_copilot/enable_ai_copilot.mdx

```diff
@@ -7,23 +7,23 @@ To use the AI Copilot, you first must agree to its terms and conditions.
 
 ## Enabling the AI Copilot
 
-After logging in to the [Migration Portal](https://migration.enterprisedb.com/) from the web browser.
-1.  Select **AI Copilot** at the top menu bar. 
-    You can also access the **AI Copilot** from the right-hand pane. This view is only available after selecting a Project and Schema.
+1.  Log in to the [Migration Portal](https://migration.enterprisedb.com/) from the web browser.
```

Comment:
I originally had written it this way, with a separate step for logging into the interface. The PM of this product, Matt, then altered it to have the login as an introductory sentence. 

@mlewandowski-edb I don't feel strongly about either, but I prefer the approach our editor Betsy is taking here, as it makes it straightforward. Also, readers tend to go straight to numbered steps and ignore the text around them. Is there a reason why you chose to make it an introductory sentence?

Reply:
I'm fine if you want to change it back to make the Login as the first step. My rationale for the change is that the copilot is a feature is a feature that is used well after logging into the migration portal. It is very unlikely that anyone will ever log into the the Migration Portal just to begin using the copilot.  That said, I'm not sure saying "After logging in to the Migration Portal ..." in a lead in sentence to the steps is much better.

What if the "Select **AI Copilot** at the top menu bar." step were changed to "Within the Migration Portal interface, select the ..."? Or, would it make sense to just omit the login step or reference to it or being in the Migration Portal?

You and @ebgitelman can make the call.